src/couch_views/src/couch_views_jobs.erl

```diff
@@ -96,7 +96,8 @@ job_data(Db, Mrst) ->
     #{
         db_name => fabric2_db:name(Db),
         ddoc_id => DDocId,
-        sig => fabric2_util:to_hex(Sig)
+        sig => fabric2_util:to_hex(Sig),
+        retries => 0
```

Comment:
This line made me think of the case where a job may have failed 2 times in the past, but has been indexing successfully since, and then encounters one more error. Should it be penalized with a failure or should we consider it "healed"... Maybe there a good place to reset `retries` to 0 after we successfully update the view.  When we finish a batch of documents...?

Reply:
Every time a new job is created with this function call will reset the retries to zero. So that means if an indexer builds an index and updates sometimes and then fails sometimes if it fails more than 3 times the build will fail. But if the build successfully completes the next time it is built it will be reset back to 0. Which I'm happy with. But I can change it to reset on every successful update?

Reply:
It was just something to think about. We don't have to do anything in this PR.

In couch replicator, for example, we have a similar issue, and what I have done there is defined a health threshold. where after a job runs successfully for so many seconds, it's considered "healthy" again.